### PR TITLE
*: remove legacy ceph_volume_client.py library

### PR DESCRIPTION
**Pull request by batrick**
This library is obsolete with the mgr volumes plugin since Nautilus.

The last remaining user of this library was Manila which will be using
the volumes plugin with Pacific and onwards.

Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -50,6 +50,12 @@
 
 * MGR: progress module can now be turned on/off, using the commands:
   ``ceph progress on`` and ``ceph progress off``.
+
+* The ceph_volume_client.py library used for manipulating legacy "volumes" in
+  CephFS is removed. All remaining users should be use the "fs volume"
```
Comment (varshar16) [inline PendingReleaseNotes:55]:
```suggestion
  CephFS is removed. All remaining users should use the "fs volume"
```

Comment (kotreshhr) [reply PendingReleaseNotes:55]: lgtm otherwise

**Review by rishabh-d-dave — COMMENTED** [review]
LGTM too otherwise.

File: PendingReleaseNotes (hunk 1 above)
Comment (batrick) [reply PendingReleaseNotes:55]: done, thanks

**Comment by varshar16** [comment]
jenkins test api

**Review by rishabh-d-dave — APPROVED** [review]

**Review by varshar16 — APPROVED** [review]

**Comment by batrick** [comment]
https://pulpito.ceph.com/pdonnell-2021-01-13_23:30:53-fs-wip-pdonnell-testing-20210113.220804-distro-basic-smithi/

**Pull request merged**